### ISSUE TITLE
Feature: integration/regression tests for transform pipelines

### ISSUE THREAD

**Issue by ajtucker** [GSS-Cogs/databaker-docker#10]
As a data engineer, I want to be able to use the latest databaker-docker image to run data transformation pipelines and rely on existing pipelines still working as expected.

Outcome: end-to-end example transform pipelines that can be run as part of a test suite in Jenkins multi-branch pipelines.

**Comment by canwaf** [comment]
Add https://github.com/GSS-Cogs/gss-local-build to the build for the dev tag, all we need to do is run `./install.sh` from `gss-local-build` and it *should* install it properly. And only spin up when requested.

**Comment by mikeAdamss** [comment]
RE integration tests, this door should be open now.

Alex recently added a flag to gssutils to reuse recordings rather than scraping data anew. See below comments.

**Comment by ajtucker** [comment]
The `RECORD_MODE` thing also records data / distributions.

**Comment by mikeAdamss** [comment]
does that include the source download from downloadURL Alex? My concern was that downloadURLs like `www.mydataset/latest.xlx` could change under us.

**Comment by ajtucker** [comment]
Yes, the session object is created when you create the scraper and if `RECORD_MODE` is set, then it should use vcrpy for all further requests and responses, including any downloads of distributions through `.as_pandas()` etc. If you set `RECORD_MODE=none`, then it should only use the pre-recorded pages and downloads.

**Comment by mikeAdamss** [comment]
awesome :smile: .thanks Alex, have updated comments accordingly.
